src/plots/layout_attributes.js

```diff
@@ -183,6 +183,7 @@ module.exports = {
         'annotations': 'annotations',
         'shapes': 'shapes',
         'images': 'images',
-        'updatemenus': 'updatemenus'
+        'updatemenus': 'updatemenus',
+        'slider': 'slider'
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6675409/18962371/8ae2737c-863e-11e6-9854-7a8193b7053e.png)

you see here, `sliders` is missing.